model/connect/PDOConnector.php

```diff
@@ -227,7 +228,7 @@ public function exec($sql, $errorLevel = E_USER_ERROR) {
 		$result = $this->pdoConnection->exec($sql);
 
 		// Check for errors
-		if ($result !== false) {
+		if (!$this->hasError($result)) {
```

Comment:
I'm not sure what you mean by the `false` return value here - this still returns the result set if the query was successful, or `null` if there was an error.

The `PDO::exec()` docs don't match with what I'm seeing - this is returning `false` with a `USE database` statement with the dblib driver, but the database select is successful.